.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Backend workflow for Golang using Snyk
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+   security::
+    runs-on: ubuntu-latest
+    strategy:
+       matrix:
+         go-version: [1.13, 1.14, 1.15, 1.x]
```

Comment:
I think we can stick with one version of go, you can check other workflows to compare

Reply:
alright. thanks